cvat/apps/dataset_manager/bindings.py

```diff
@@ -1129,6 +1131,8 @@ def convert_attrs(label, cvat_attrs):
         anno_label = map_label(shape_obj.label)
         anno_attr = convert_attrs(shape_obj.label, shape_obj.attributes)
         anno_attr['occluded'] = shape_obj.occluded
+        if shape_obj.type == ShapeType.RECTANGLE:
+            anno_attr['rotation'] = shape_obj.rotation
```

Comment:
The main question is. Should we rewrite existing ``rotation`` attribute? (if it exists)

Reply:
Maybe, we need to have a general policy about this, which also includes `occluded` and other things.

Reply:
``occluded``, ``outside`` existed always with CVAT. But rotation does not and a existing tasks can include this attribute.